charts/flink-job/templates/flinkjobcluster.yaml

```diff
@@ -53,7 +53,19 @@ spec:
       {{- toYaml . | nindent 6 }}
       {{- end }}
     {{- end }}
-  # volumeClaimTemplates, initContainers, nodeSelector, tolerations, sideCars not exposed
+    # volumeClaimTemplates, initContainers, sideCars not exposed
+    {{- if .Values.nodeSelector }}
```

Comment:
Make sure to edit `values.yaml` and add the these ettings to the file, along with doc strings...